packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -67,13 +42,7 @@ class SystemRulesTable extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            columns: [
-                { title: 'Rule', transforms: [ sortable ] },
-                { title: 'Policy', transforms: [ sortable ] },
-                { title: 'Severity', transforms: [ sortable ] },
-                { title: 'Passed', transforms: [ sortable ] },
-                { title: <React.Fragment>{ ANSIBLE_ICON } Ansible</React.Fragment>, transforms: [ sortable ] }
-            ],
+            columns: props.columns,
```

Comment:
The `columns` prop seems to be missing a type validation in the prop-types. It might be good to verify the shape of each `column` item as well. and possibly a default value.